config/betterquesting/DefaultQuests/Quests/AndSoItBegins-AAAAAAAAAAAAAAAAAAAAAA==/CaughtintheWeb-AAAAAAAAAAAAAAAAAAAHMw==.json

```diff
@@ -31,7 +31,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "Spiders to watch for...\n\nWitch spiders reflect missile damage back at you. Look for the red X on their back, and draw your melee weapon and prepare for close combat.\n\nMother spiders will spawn a lot of baby spiders, some of which may be infernal.\n\nA flying spider can grab the oblivious flyer.\n\nGhost spiders are partially invisible and can pass through thin walls."
+      "desc:8": "Some spider have nasty modifiers:\n\nWitch spiders reflect missile damage back at you. Look for the red X on their back and use your melee weapon.\n\nMother spiders will spawn a lot of baby spiders on death, some of which may be infernal.\n\nA flying spider can grab the oblivious flier.\n\nGhost spiders are partially invisible and can pass through thin walls."
```

Comment:
"draw your melee weapon and prepare for close combat" sounds great why remove that?

Reply:
Thank you. Fixed.